examples/core/inject_lib_blob/src/main.rs

```diff
@@ -0,0 +1,25 @@
+use frida::{Frida, Inject};
+use lazy_static::lazy_static;
```

Comment:
There are alternatives in the rust stdlib these days, no reason to use lazy_static anymore, such as:
https://doc.rust-lang.org/std/cell/struct.LazyCell.html

Reply:
Thank you guys. I didn't know. Changes made.